beads_web/src/main/java/com/beads/web/dao/SearchCriteria.java

```diff
@@ -0,0 +1,86 @@
+package com.beads.web.dao;
+
+import com.beads.model.domain.OrderStatus;
+import java.time.LocalDateTime;
+
+public class SearchCriteria {
+
+  private Integer orderId;
+  private String email;
+  private OrderStatus status;
+  private String phoneNumber;
+  private LocalDateTime dateOfOrder;
+  private String address;
+
+  public boolean isOrderIdNotNull() {
+    return orderId != null;
+  }
+
+  public boolean isEmailNotNull() {
+    return email != null && !email.isEmpty();
+  }
+
+  public boolean isStatusNotNull() {
+    return status != null && !status.getDbValue().isEmpty();
```

Comment:
if `status != null` `getDbValue()` will always return nonempty value
so only check `status != null` will be ok